cmd/svcat/plan/get_cmd.go

```diff
@@ -90,23 +100,46 @@ func (c *getCmd) getAll() error {
 		return fmt.Errorf("unable to list classes (%s)", err)
 	}
 
+	if c.classFilter != "" {
```

Comment:
The design of the svcat command is such that only cli concerns (formatting console output for example) is in this cmd/svcat package. All other logic is handled in the svcat package (/pkg/svcat) so that it's exposed in our library.

The call above to `plans, err := c.App.RetrievePlans()` should look something like this instead:

```golang
// The naming convention in the svcat sdk is ID = the k8s uuid and Name = the external name that we all prefer
opts := svcat.RetrieveOptions { ClassID: "CLASS_UUID", ClassName: "CLASS_EXT_NAME" } 
plans, err := c.App.RetrievePlans(opts)
```